src/lib/utils/entry-point.ts

```diff
@@ -384,7 +391,23 @@ function getEntryPointsForPackages(
         }
 
         results.push({
-            displayName: packageJson["name"] as string,
+            displayName:
+                typedocPackageConfig?.displayName ??
+                // if displayName is not configured, use the package name (and version, if configured)
+                `${packageJson["name"]}${
+                    includeVersion && packageJson["version"]
+                        ? `@${packageJson["version"]}`
+                        : ""
+                }`,
+            readmeFile: typedocPackageConfig?.readmeFile
```

Comment:
Any reason not to default this to `README.md` in the same directory as the `package.json`?... in fact, do you have any examples in the wild where this would *not* be the (case insensitive) desired file to read?

Reply:
Yeah, I base this off the `readme` field in `typedoc` config. I think the default there is `none`. Also, when I think of this feature, one use case that makes sense to me is that we can have a directory somewhere with a bunch of markdown files, and the `readmeFile` link per child package can point of those files instead of the `README.md` file in each pacakge directory. That way, I can collate documents to some folder rather than having to scatter them in each package, for example:


```
documentation
  - doc1.md
  - doc2.md
packages
  - pkg1
    - package.json: typedoc.readmeFile --> ../../documentation/doc1.md
  - pkg2
```

Well, the one thing that I'm concerned about is the name of the config `readmeFile`, which seems to mislead people into thinking that this is pointing at the `README`, the way I think about this is more like a custom doc page per package.

Reply:
> I think the default there is none

The default is to discover a readme file starting at the common directory of all source files and working up the file system.

> the way I think about this is more like a custom doc page per package.

The reason I'm willing to let this in today is *because* it's a README file for a package and is intentionally limited in scope. The big feature for 0.24 is planned to be arbitrary extra markdown files in a project, but this is definitely not that. `readmeFile` is a good name IMO.

Reply:
so I guess I don't need to do anything here then? I'm going ahead to resolve this.